server/src/main/java/org/opensearch/repositories/blobstore/BlobStoreRepository.java

```diff
@@ -1399,9 +1538,10 @@ private void executeOneStaleIndexDelete(
                 } catch (Exception e) {
                     assert false : e;
                     logger.warn(new ParameterizedMessage("[{}] Exception during single stale index delete", metadata.name()), e);
+                    listener.onFailure(e);
```

Comment:
can we remove this as the failure while releasing the lock file will be caught by this same `catch` block and index folder deletion will not happen in those cases. So we should be fine ignoring this failure as otherwise failure in the stale index folder deletion will also fail the delete snapshot calls.

Reply:
Removed in the latest revision